examples/event_handling/lasso_demo.py

```diff
@@ -38,9 +38,8 @@ def __init__(self, ax, data):
 
         facecolors = [d.color for d in data]
         self.xys = [(d.x, d.y) for d in data]
-        fig = ax.figure
         self.collection = RegularPolyCollection(
-            fig.dpi, 6, sizes=(100,),
+            6, sizes=(100,),
```

Comment:
`6` -> rotation angle; not sure if the `fig.dpi` was intended to produce perfect circles and the `6` should have been deleted instead.